.github/workflows/codeql.yml

```diff
@@ -0,0 +1,77 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '21 22 * * 1'
```

Comment:
Can you add a human-readable comment for this?

Reply:
I added in the comment. Everything within this workflow is barebones defaults.

By the way, GitHub automatically translates crons to English when viewing via web

![image](https://github.com/reactive-python/reactpy-django/assets/16909269/1b42df60-a8e0-4e56-8283-d7efb4479f70)